src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -496,7 +496,8 @@ Namespace Microsoft.CodeAnalysis.Semantics
         Private Function CreateBoundTryCastOperation(boundTryCast As BoundTryCast) As IConversionExpression
             Dim operand As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() Create(boundTryCast.Operand))
             Dim syntax As SyntaxNode = boundTryCast.Syntax
-            Dim conversion As Conversion = _semanticModel.GetConversion(syntax)
+            Dim summary As BoundNodeSummary = New BoundNodeSummary(boundTryCast.Operand, boundTryCast, Nothing)
+            Dim conversion As Conversion = _semanticModel.GetTypeInfoForNode(summary).ImplicitConversion
```

Comment:
> I'll move it to use Conversions.ClassifyConversion

I believe all we need is already in the bound node. We don't need to reclassify conversion. Especially, we shouldn't use ClassifyConversions for TryCast/DirectCast.